winter/routing/routing.py

```diff
@@ -40,16 +39,28 @@ def route_put(url_path='', produces: Optional[Tuple[MediaType]] = None, consumes
     return route(url_path, 'PUT', produces=produces, consumes=consumes)
 
 
-def register_route(
-        func,
-        url_path: str,
-        http_method: Optional[str],
-        produces: Optional[Tuple[MediaType]],
-        consumes: Optional[Tuple[MediaType]],
-):
-    assert func not in _route_annotations, f'{func} is already mapped to a route'
-    _route_annotations[func] = RouteAnnotation(url_path, http_method, produces, consumes)
-
-
-def get_route_annotation(func) -> Optional[RouteAnnotation]:
-    return _route_annotations.get(func)
+def get_route_annotation(component_method_or_component) -> Optional[RouteAnnotation]:
```

Comment:
Let's inline the simple wrapper?

Reply:
I have delete this function at all
I think we do not need it

Reply:
Nice, that's what I wanted.